### ISSUE TITLE
Ability to disable SSL certificate verification

### ISSUE THREAD

**Issue by darrylhebbes** [baohaojun/org-jira#28]
I work with a server where the certificate is self signed and cannot connect Org-Jira to this instance.
I have a similar script written in Python which does work if I disable SSL certificate verification.

**Comment by baohaojun** [comment]
I tested with a Jira server on localhost with ssl enabled according to https://confluence.atlassian.com/display/JIRA/Integrating+JIRA+with+Apache+using+SSL, there is no problem getting back the issues. Except there is a message saying: "gnutls.c: [0] (Emacs) fatal error: The TLS connection was non-properly terminated." But there is no problem operating on the issues (get, step the workflow, updating the description).

Then when I tried again, it stopped working at all, no issues can be updated, or comments fetched. Every time there is that gnutls.c error message. 

So this seems to be a Emacs+Gnutls problem, as there are plenty discussion about this: https://github.com/nicferrier/elmarmalade/issues/55

I will track this further, see if I can work around the current gnutls error.
